system/ee/ExpressionEngine/Controller/Members/Roles/Roles.php

```diff
@@ -403,21 +403,33 @@ public function edit($id)
 
     private function setWithPost(Role $role)
     {
+        $site_id = ee()->config->item('site_id');
+        
         $role_groups = !empty(ee('Request')->post('role_groups')) ? ee('Request')->post('role_groups') : array();
 
         $role->name = ee('Request')->post('name');
         $role->short_name = ee('Request')->post('short_name');
         $role->description = ee('Request')->post('description');
 
-        //We don't allow much editing for SuperAdmin role, so just enfore it's locked and return here
+        //We don't allow much editing for SuperAdmin role, so just enforce it's locked and return here
         if ($role->getId() == 1) {
             $role->is_locked = 'y';
             return $role;
         }
         $role->is_locked = ee('Request')->post('is_locked');
         $role->RoleGroups = ee('Model')->get('RoleGroup', $role_groups)->all();
         $role->AssignedModules = ee('Model')->get('Module', ee('Request')->post('addons_access'))->all();
-        $role->AssignedUploadDestinations = ee('Model')->get('UploadDestination', ee('Request')->post('upload_destination_access'))->all();
+
+        $upload_destination_ids = !empty(ee('Request')->post('upload_destination_access')) ? ee('Request')->post('upload_destination_access') : array();
```

Comment:
I know were not PSR'ing all variable names I think it would be good to PSR this one since the other variable names added are PSR'ed